.semaphore/semaphore.yml

```diff
@@ -35,13 +37,26 @@ blocks:
             - make VERBOSE=1
         - name: Clang-9 Build
           env_vars:
+            - name: COMPILER
+              value: clang
             - name: CC
               value: clang-9
             - name: CXX
               value: clang++-9
           commands:
             - mkdir build.clang && cd build.clang && cmake -DENABLE_WEBSERVER=ON -DCMAKE_VERBOSE_MAKEFILE=1 -DENABLE_WEBCAM=ON ..
             - make VERBOSE=1
```

Comment:
Any chance that a `make -j$(nproc --all) VERBOSE=1` would work here? Or would Semaphore malfunction?

Reply:
i guess that should work. Let's try it in a separate pr